### DISCUSSION TITLE
Uploading translator notes in Glossaries

### DISCUSSION THREAD

**Discussion by jpujolar** [WeblateOrg/weblate#10760] [Q&A]
I have noted in the documentation that, if you wish to upload a file with glossary entries, most uploading options only take up the English term and the translated term, and one upload option also includes whatever is in the key field.

Is there a way to upload translator notes too? My idea was to include plenty of info for other translators in this field.

Best

Joan

**Comment by nijel** [comment]
Yes, it was just lacking documentation, I've updated it in 52008ff3eb41e2d606e08d00560e659c401bd170. 

It should shortly also appear at https://docs.weblate.org/en/latest/formats/tbx.html
